pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -564,3 +564,10 @@ def test_dt_subclass_add_timedelta(lh, rh):
     result = lh + rh
     expected = SubDatetime(2000, 1, 1, 1)
     assert result == expected
+
+
+def test_normalize_pre_epoch_dates():
```

Comment:
in timestamp.test_unary_ops there is a test_normalize this can go right below

Reply:
Thanks, moved the test